lib/webdriveragent.js

```diff
@@ -121,6 +121,13 @@ class WebDriverAgent {
     return !!(await this.getStatus());
   }
 
+  getBasePath () {
+    if (this.url.path === '/') {
+      return '';
+    }
+    return this.url.path;
```

Comment:
should it be `this.url.path || ''`?

Reply:
hm yes.
`url.parse('')` returns `null` as `path`. will revert 6b70c74